src/pages-and-resources/__snapshots__/SettingsComponent.test.jsx.snap

```diff
@@ -1,3 +0,0 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
-
-exports[`SettingsComponent renders LazyLoadedComponent when provided with props 1`] = `[Function]`;
```

Comment:
This snapshot test was not implemented properly - the snapshot was just `[Function]`